commands/profile.go

```diff
@@ -179,8 +182,10 @@ func init() {
 		"\nIf security is disabled, provide --auth-type='disabled'.\nIf security uses HTTP basic authentication, provide --auth-type='basic'.\n"+
 		"If security uses AWS IAM ARNs as users, provide --auth-type='aws-iam'.\nodfe-cli asks for additional information based on your choice of authentication type.")
 	_ = createProfileCmd.MarkFlagRequired(FlagProfileCreateAuthType)
-	createProfileCmd.Flags().IntP(FlagProfileMaxRetry, "m", 3, "Specifies a value of maximum retry attempts the odfe-cli retry handler can perform.\n"+
+	createProfileCmd.Flags().IntP(FlagProfileMaxRetry, "m", 3, "Maximum retry attempts allowed if transient problems occur.\n"+
 		"You can override this value by using the ODFE_MAX_RETRY environment variable.")
+	createProfileCmd.Flags().Int64P(FlagProfileTimeout, "t", 60, "Maximum time allowed for connection in seconds.\n"+
```

Comment:
Why 60 second default? Wouldnt 10 be better?

Reply:
updated to 10s